pkg/api/middleware.go

```diff
@@ -41,19 +41,10 @@ func (api *API) recoverPanic(next http.Handler) http.Handler {
 
 func (api *API) handleCORS(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		origin := r.Header.Get("Origin")
-		var allowedHost string
-		allowedHosts := []string{"https://www.ddstats.com", "http://www.ddstats.com", "http://ddstats.com", "https://ddstats.com"}
```

Comment:
can you update this to `allowedHosts := []string{"*"}` instead

Reply:
i believe using that method would be comparing request hostname against origin named "*", which would always fail because it's a literal comparison, hence we're setting the ACAO header directly to "*" which is properly treated like a wildcard